### DISCUSSION TITLE
Statistics Graph: Allow entities with two unique units of measurement

### DISCUSSION THREAD

**Discussion by rkost** [home-assistant/frontend#15416] [Dashboard]
If I understand it correctly, the *Statistics Graph* Card is currently restricted to only support entities with the same unit of measurement properly. Selecting e.g. a `humidity entity` and a `temperature entity` in the same card will result in something like this:

![image](https://user-images.githubusercontent.com/11631510/218073392-6fafcca2-3817-43b4-93e5-c3b9b662e643.png)

However it would be nice to be able to compare these two entities in a more compact style, like you see in so called *Climographs*. One scale goes to the left (like in the current implementation of the card) and the other one goes to the right side of the graph. Doing it this way would allow both entities to scale to the available vertical space. Example:

![image](https://user-images.githubusercontent.com/11631510/218073871-8a34e509-fdca-4c5d-8224-be746154ef91.png)

It might make the card too complicated so maybe we would want this option as a separate card? What do you think?

PS: This is a similar discussion, but is not quite my point: https://github.com/home-assistant/frontend/discussions/8840

**Comment by fischer-felix** [comment]
Would love to see that too.

Since it's been quite a while, have you found any alternative to use in the meantime?

**Reply by rkost** [reply]
Sadly no. I gave up on that topic and never re-visited it again since. Still bothers me though.

**Reply by fischer-felix** [reply]
Upon further searching I have actually found the [apexcharts-card](https://github.com/RomRider/apexcharts-card) to work quite well for this.
The amount of configuration options is pretty much endless (although that also makes it less easy to set up).
